easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -318,8 +318,29 @@ def configure_step(self, srcdir=None, builddir=None):
 
         (out, _) = run_cmd(command, log_all=True, simple=False)
 
-        return out
-
+        # sanitycheck for the configuration step
+        self.log.info("Checking python paths")
+        with open('CMakeCache.txt', 'r') as file:
+            lines = file.readlines()
+
+        # Search for Python paths in each line
+        for line in lines:
+            if line.startswith('Python3_EXECUTABLE:FILEPATH='):
```

Comment:
It looks like these are only guaranteed for CMake 3.16+ so should probably make this check conditional on that version

Reply:
They also don't cover the FindPython case: https://cmake.org/cmake/help/latest/module/FindPython.html#artifacts-specification

Reply:
Yes maybe use a regex? `Python[2-3]?_EXECUTABLE`?